### ISSUE TITLE
New diagnostic: The Values attribute can be simplified.

### ISSUE THREAD

**Issue by Bartleby2718** [nunit/nunit.analyzers#755]
As mentioned in https://docs.nunit.org/articles/nunit/writing-tests/attributes/values.html#values-with-enum-or-boolean
```cs
[Test]
public void MyBoolTest([Values(true, false)] bool value) { /* ... */ } 
```
can be simplified to
```cs
[Test]
public void MyBoolTest([Values] bool value) { /* ... */ } 
```

We could add a new diagnostic to detect and simplify this.

The same goes for nullable boolean and enums.

**Comment by Bartleby2718** [comment]
I can take a stab at this, so please assign this to me!

**Comment by manfred-brands** [comment]
> I can take a stab at this, so please assign this to me!

Done. Go ahead.

**Comment by mikkelbu** [comment]
Fixed by #758
